src/modules/product/product.route.js

```
@@ -1,20 +1,68 @@
 import { cacheMiddleware } from "../../common/middlewares/cache.middleware.js";
+// import { authMiddleware } from "../../common/middlewares/auth.middleware.js";
+import multer from 'multer';
+import path from 'path';
+
+// Configure multer storage
+const storage = multer.diskStorage({
+  destination: function (req, file, cb) {
+    cb(null, './uploads/temp'); // Temporary storage before Cloudinary upload
+  },
+  filename: function (req, file, cb) {
+    const uniqueSuffix = Date.now() + '-' + Math.round(Math.random() * 1E9);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)